string object_id
uint32 errno
